docs/content/toc.md

```diff
@@ -164,10 +165,8 @@ layout: toc
   * Experimental Features
     * [Overview](/docs/VERSION/development/experimental.html)
     * [Approximate Histograms and Quantiles](/docs/VERSION/development/extensions-core/approximate-histograms.html)
-    * [Datasketches](/docs/VERSION/development/extensions-core/datasketches-extension.html)
     * [Geographic Queries](/docs/VERSION/development/geo.html)
     * [Router](/docs/VERSION/development/router.html)
-    * [Kafka Indexing Service](/docs/VERSION/development/extensions-core/kafka-ingestion.html)
```

Comment:
it would be nice for these items to remain in the ToC for faster access

Reply:
KIS is already in the ingestion section.

DataSketches isn't in the ToC but it's described in the approximate algorithms and sql sections.

Actually, now that I think about this a bit more, I think removing the entire experimental feature section is a good idea. There is no particular reason to have them displayed so prominently. It is better for them to be in the sections that make sense for them (node type, aggregator, whatever).